lib/PromptSession.ts

```diff
@@ -444,17 +458,116 @@ export class PromptSession {
 	 * Generates a list of options for chooseActionLoop
 	 * @param projectLibrary to generate options for
 	 */
-	private generateActionChoices(projectLibrary: ProjectLibrary): string[] {
-		const actionChoices: string[] = ["Complete & Run"];
+	private generateActionChoices(projectLibrary: ProjectLibrary): Array<{}> {
+		const actionChoices: Array<{}> = [{
+			name: "Complete & Run" + chalk.gray("..........install packages and run in the default browser"),
+			short: "Complete & Run",
+			value: "Complete & Run"
+		}];
 		if (projectLibrary.components.length > 0) {
-			actionChoices.push("Add component");
+			actionChoices.push({
+				name:  "Add component" + chalk.gray("...........add a specific component view (e.g a grid)"),
+				short: "Add component", // displayed result after selection
+				value: "Add component" // actual selection value
+			});
 		}
 		if (projectLibrary.getCustomTemplateNames().length > 0) {
-			actionChoices.push("Add view");
+			actionChoices.push({
+				name: "Add scenario " + chalk.gray("...........add a predefined scenario view (e.g grid or dashboard)"),
+				short: "Add scenario",
+				value: "Add scenario"
+			});
 		}
 
 		return actionChoices;
 	}
+
+	/**
+	 * Generates a list of options for addComponent
+	 * @param projectLibrary to generate options for
+	 */
+	private generateGroupChoices(projectLibrary: ProjectLibrary): Array<{}> {
+		const groups = projectLibrary.getComponentGroups();
+		const groupChoices: Array<{}> = [];
+		for (const group of groups) {
+			groupChoices.push({
+				name: group + chalk.gray(`...........contains similar ${group} components`),
```

Comment:
@bazal4o We might need to add an interface for the groups so they can.. have actual descriptions.
That on it's own is a pretty decent chunk of extra changes.. 😒